sns_driver.go

```diff
@@ -0,0 +1,93 @@
+package es
+
+import (
+	"encoding/json"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/sns"
+)
+
+// NewSNSDriver creates an SNSDriver
+func NewSNSDriver(client *sns.SNS, topicArn string, driver Driver) *SNSDriver {
+	return &SNSDriver{
+		client:   client,
+		topicArn: topicArn,
+		driver:   driver,
+	}
+}
+
+// SNSDriver creates a driver decorator that sends a notification to AWS' SNS
+// when new events are saved.
+type SNSDriver struct {
+	client   *sns.SNS
+	topicArn string
+	driver   Driver
+}
```

Comment:
I like the decorator pattern but would we expose both PG and SNS drivers or could we possibly expose only one as the only way of handling events?

Reply:
The idea I had in mind is that eventually we'll have one "do-it-all" kind of driver, but composed by smaller pieces…